README.md

```diff
@@ -83,37 +83,264 @@ pip install sparsezoo
 
 ## Quick Tour
 
-### Python APIs
+The SparseZoo Python API enables you to search and download models. Code examples are given below.
+We encourage the users to load models by copying a stub directly from a [model page]((https://sparsezoo.neuralmagic.com/)).
 
-The Python APIs respect this format enabling you to search and download models. Some code examples are given below.
-The [SparseZoo UI](https://sparsezoo.neuralmagic.com/) also enables users to load models by copying
-a stub directly from a model page.
+### Introduction to Model Class Object
 
+The `Model` is a fundamental object that serves as a main interface with the SparseZoo library. 
+It represents a SparseZoo model, together with all its directories and files.
 
-#### Loading from a Stub
+#### Creating a Model Class Object From SparseZoo Stub
+```python
+from sparsezoo import Model
+
+stub = "zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95_quant-none"
+
+model = Model(stub)
+print(str(model))
+
+>> Model(stub=zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95_quant-none)
+```
+
+#### Creating a Model Class Object From Local Model Directory
+```python
+from sparsezoo import Model
+
+directory = ".../.cache/sparsezoo/eb977dae-2454-471b-9870-4cf38074acf0"
+
+model = Model(directory)
+print(str(model))
+
+>> Model(directory=.../.cache/sparsezoo/eb977dae-2454-471b-9870-4cf38074acf0)
+```
+
+#### Manually Specifying the Model Download Path
+
+Unless specified otherwise, the model created from the SparseZoo stub is saved to the local sparsezoo cache directory. 
+This can be overridden by passing the optional `download_path` argument to the constructor:
+
+```python
+from sparsezoo import Model
+
+stub = "zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95_quant-none"
+download_directory = "./model_download_directory"
+
+model = Model(stub, download_path = download_directory)
+
+```
+#### Downloading the Model Files
+Once the model is initialized from a stub, it may be downloaded either by using the `download()` method or by calling a `path` property. Both pathways are universal for all the files in SparseZoo. Calling the `path` property will always trigger file download unless the file has already been downloaded.
```

Comment:
```suggestion
Once the model is initialized from a stub, it may be downloaded either by calling the `download()` method or by invoking a `path` property. Both pathways are universal for all the files in SparseZoo. Invoking the `path` property will always trigger file download unless the file has already been downloaded.
```